analysis_options.yaml

```diff
@@ -19,45 +19,19 @@ analyzer:
     - 'testing/test_package_export_error/**'
 linter:
   rules:
-    - always_declare_return_types
-    - avoid_dynamic_calls
-    - avoid_single_cascade_in_expression_statements
-    - avoid_unused_constructor_parameters
-    - annotate_overrides
-    - avoid_init_to_null
-    - directives_ordering
-    - no_adjacent_strings_in_list
-    - package_api_docs
-    - prefer_final_fields
-    - prefer_initializing_formals
-    - prefer_void_to_null
-    - slash_for_doc_comments
-    - type_annotate_public_apis
-    #    - unnecessary_brace_in_string_interps
+    always_declare_return_types: true
```

Comment:
Because there is no way to say "false" in a list (unless there's something I'm missing), to override something in recommended/core.  If I want to say false later I will have to convert to a map, so just do it now.

Reply:
Yeah.  There's a note about this short-coming in the docs:

https://dart.dev/guides/language/analysis-options#disabling-individual-rules

Unfortunate.